app/pages/project/classify.cjsx

```diff
@@ -305,9 +305,25 @@ module.exports = React.createClass
 
     @loadAppropriateClassification(@props) if @props.workflow?
 
+  getPromotionMessage: (props) ->
```

Comment:
Could this be set in [the translations wrapper for projects](/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/project-translations.jsx), then referenced as `props.translation.promotion_message`:

Reply:
I've moved the default promotion message into the project translation wrapper. I'm hesitant to put the custom messages for Gravity Spy and Snapshot Wisconsin in there because these being hard coded is temporary until it gets moved into the project contents in Panoptes. Once those are there, we can check for the presence of those project content strings or fallback to the default messaging. Does this work?

Reply:
The translation wrapper should take care of that fallback for you, shouldn't it?

I don't think it makes a big difference here, but I'd prefer if components expect their language strings to be provided externally, via the `translation` prop, rather than defining them themselves.

Reply:
Ok, I can do another quick refactor to have the handling of this in the translation wrapper.